lua/tailwindcss-colorizer-cmp/init.lua

```diff
@@ -84,12 +84,28 @@ M.formatter = function(entry, vim_item)
 			table.insert(words, word)
 		end
 
+		local color_name, color_number
+		if
+			words[2] == "x"
+			or words[2] == "y"
+			or words[2] == "t"
+			or words[2] == "b"
+			or words[2] == "l"
+			or words[2] == "r"
+		then
+			color_name = words[3]
+			color_number = words[4]
+		else
+			color_name = words[2]
+			color_number = words[3]
+		end
+
 		-- FIXME:
 		-- handle special case of stuff like whatever-black/white
 		-- are there other cases like foo-bar-white
-		if #words == 2 and (words[2] == "white" or words[2] == "black") then
+		if color_name == "white" or color_name == "black" then
```

Comment:
Thanks for your changes, looking good!

What happens if the color is `something-x-{black|white}`? `words` will be `#3`, how does that affect the behaviour of this change, and in general, is black or white being the third param currently supported? You may be fixing two issues here! I'm asking since I haven't tested your change and I'm not sure.. happy for you to do some more testing or I can have a look when I have some time.

Again, I appreciate your effort on this.

Reply:
Whoops my bad, with this change, words can be 2 OR 3 here. `something-x-{black|white}` is what I'm looking to support here. Since in tailwind black and white never have any numbers on them, `#words` will always be 2 (or 3 if they have the second prefix), so the check isn't necessary.

There is an edge case though. Let's say someone has a custom tailwind color of `foo-black-bar`, it'll just return it as black, as we're not checking the length.

Reply:
So something like `border-black` still works? If so, we can merge!